freqtrade/worker.py

```diff
@@ -63,60 +67,74 @@ def run(self) -> None:
             if state == State.RELOAD_CONF:
                 self._reconfigure()
 
-    def _worker(self, old_state: Optional[State], throttle_secs: Optional[float] = None) -> State:
+    def _worker(self, old_state: Optional[State]) -> State:
         """
         Trading routine that must be run at each loop
```

Comment:
maybe reword this to 
```suggestion
        Main routine that must be run at each loop
```
?